### PR TITLE
Convert CSS to TailwindCSS 3.x

### PR DESCRIPTION
**Pull request by Jackardios**
This PR adds CSS to TailwindCSS 3.x converter. ([old version](https://github.com/ritz078/transform/pull/235) only support 1.x version of tailwindcss)

https://github.com/Jackardios/css-to-tailwindcss/ package is used ([VSCode extension](https://github.com/Jackardios/vscode-css-to-tailwindcss) also available)

### Features:
- supports all (or almost all) the features currently available in TailwindCSS
- supports nested sass-like syntax (via postcss-nested)
- the ability to set your own TailwindCSS configuration
- colors are matched regardless of the format used
- rem is converted to px (it is possible to configure the rem size)
- non-convertible CSS declarations are simply skipped
- [ambiguities](https://tailwindcss.com/docs/adding-custom-styles#resolving-ambiguities) when using css variables are resolved automatically

### CODE REVIEW

**Comment by ritz078** [comment]
@Jackardios Thank you for this. Can you update the existing `css-to-tailwindcss` itself instead of adding a new route for version 3. v1 is very old and in the long run it's hard to maintain one route per version.

**Comment by Jackardios** [comment]
@ritz078 done!

**Review by ritz078 — APPROVED** [review]

File: pages/css-to-tailwind.tsx [hunk 1]
```diff
@@ -70,6 +72,17 @@ function CssToTailwindSettings({
...
+          intent="none"
+          title={
+            <>
+              <Link href="https://marketplace.visualstudio.com/items?itemName=Jackardios.vscode-css-to-tailwindcss">
+                VSCode extension also available
+              </Link>
+            </>
+          }
+          marginBottom={20}
+        />
```
Comment (ritz078) [inline pages/css-to-tailwind.tsx:85]: @Jackardios I appreciate the contribution, but cannot treat one package differently than others. There is a link to the npm package for examples that use a package. I am open for any feedback for linking back to the original source but that should be applicable for examples.

I hope you understand.

Comment (Jackardios) [reply pages/css-to-tailwind.tsx:85]: Ok, no problem!

**Comment by ritz078** [comment]
@allcontributors please add @Jackardios for code.

**Comment by craig-helpshift** [comment]
Does this mean that https://transform.tools/css-to-tailwind will support tailwind 3.0 configs now?

**Comment by Jackardios** [comment]
@craig-helpshift Yes (but configurations must not include import or require calls)

**Comment by craig-helpshift** [comment]
@Jackardios Ok, the reason I asked is because I tried the custom config at this playground: https://play.tailwindcss.com/XF2wTG5BVg

And it didn't seem to work in transform.tools. Any idea why?

**Comment by Jackardios** [comment]
@craig-helpshift Yes, `prefix`, `separator` and `corePlugins` parameters  don't work yet, but I fixed it all at https://github.com/ritz078/transform/pull/355

**Pull request merged**